### ISSUE TITLE
Broken link on fstar-lang.org site, "Talks" section

### ISSUE THREAD

**Issue by valera-rozuvan** [FStarLang/FStar#1552]
In the section [Talks](https://www.fstar-lang.org/#talks), there is a sub-section with 2 links:

```
What's new in F*?
  - Recent: Talk at Everest Workshop: slides (2017-10-02)
```

Both links lead to incorrect resources. The `slides` link should point to [http://prosecco.gforge.inria.fr/personal/hritcu/talks/fstar_all_hands_sept_2017/out/fstar-all-hands.html](http://prosecco.gforge.inria.fr/personal/hritcu/talks/fstar_all_hands_sept_2017/out/fstar-all-hands.html).

**Comment by mtzguido** [comment]
Yes, it is private, and also contains many more things. Perhaps we should have a separate web repo.
